config/install/core.entity_view_display.node.alert_banner.default.yml

```
@@ -2,35 +2,34 @@ langcode: en
 status: true
 dependencies:
   config:
-    - field.field.node.govcms_alert.field_alert_date
-    - field.field.node.govcms_alert.field_alert_display_date
-    - field.field.node.govcms_alert.field_alert_link
-    - field.field.node.govcms_alert.field_alert_message
-    - field.field.node.govcms_alert.field_alert_page_visibility
-    - field.field.node.govcms_alert.field_alert_permanent
-    - field.field.node.govcms_alert.field_alert_type
-    - node.type.govcms_alert
+    - field.field.node.alert_banner.field_alert_display_date
+    - field.field.node.alert_banner.field_alert_link
+    - field.field.node.alert_banner.field_alert_message
+    - field.field.node.alert_banner.field_alert_page_visibility
+    - field.field.node.alert_banner.field_alert_permanent
+    - field.field.node.alert_banner.field_alert_type
+    - node.type.alert_banner
   module:
     - datetime
     - link
     - text
     - user
-id: node.govcms_alert.default
+id: node.alert_banner.default
 targetEntityType: node
-bundle: govcms_alert
+bundle: alert_banner
 mode: default
 content:
   field_alert_display_date:
-    type: datetime_default
-    weight: 3
-    region: content
+    weight: 4
     label: above
     settings:
+      format_type: medium
```

```suggestion
      format_type: alert_banner_date
```
@govindmaloo the Alert Display Date must be displayed with the date format provided by the module, originally `govcms_alert_date`.

---

@sonnykt Updated